packages/db/lib/gen-schema.mjs

```diff
@@ -52,4 +54,10 @@ function printOpenAPISchema (_args) {
   })
 }
 
-export { printGraphQLSchema, printOpenAPISchema }
+const filenameConfigJsonSchema = 'platformatic.db.schema.json'
+
+async function generateJsonSchemaConfig () {
+  await writeFile(filenameConfigJsonSchema, JSON.stringify(platformaticDBschema, null, 2))
+}
```

Comment:
Can you add a test for this too?

Reply:
[c42152c](https://github.com/platformatic/platformatic/pull/418/commits/c42152c4727409d91031f04a9b0dfa120eb9aa95)